### ISSUE TITLE
Rule to limit variable statements to a single declaration

### ISSUE THREAD

**Issue by DanielRosenwasser** [palantir/tslint#525]
It would be nice if we could disallow the following in our codebase:

```TypeScript
var x: string, y: number;
```

**Comment by ashwinr** [comment]
to clairfy: do you mean disallow multiple variable definitions in the same statement?

**Comment by DanielRosenwasser** [comment]
Yup, sorry about that. It was fairly late.

**Comment by adidahiya** [comment]
Looks like the corresponding eslint rule for this is called `one-var`: http://eslint.org/docs/rules/one-var

**Comment by ashwinr** [comment]
:+1:

**Comment by YuichiNukiyama** [comment]
related PR: #1194

**Comment by JKillian** [comment]
Decided not to go with `one-var` as it's a somewhat brief name, instead we'll use `no-multiple-variable-declaration`, which fits TSLint's style better hopefully. Thanks to @YuichiNukiyama for implementing this! It'll be around in the next release. (Still up for renaming things if people feel strongly about it one way or the other)

**Comment by JKillian** [comment]
Actually, I think `no-multiple-variable-declaration` is ambiguous in which style it's actually disallowing. Now thinking about `one-variable-per-declaration` instead
